src/components/Operations.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+
+class Operations extends Component {
+    constructor() {
+        super()
+        this.state = {
+            amountInput: '',
+            vendorInput: '',
+            categoryInput: '',
+        }
+    }
+
+    handleCategory = (e) => this.setState({ categoryInput: e.target.value })
```

Comment:
you should use one input handler. give inputs a name then you can grab the name `e.target.name`

`handleInput = (e) => this.setState({ [e.target.name]: e.target.value })`